lib/prepare.test.js

```diff
@@ -146,6 +146,47 @@ it("Update lerna.json and root package.json when one or more package has changed
 	expect(context.log).toHaveBeenCalledWith("Write version %s to lerna.json in %s", "1.0.0", cwd);
 });
 
+it("Update only lerna.json when one or more package has changed", async () => {
```

Comment:
Maybe this description should contain something about `rootVersion` being `false`?